### PR TITLE
[Azure Analysis Services] Replace backup from configuration object to container Uri string

### PR DESCRIPTION
**Pull request by taiwu**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: arm-analysisservices/2016-05-16/swagger/analysisservices.json [hunk 1]
```diff
@@ -519,8 +519,9 @@
         "asAdministrators": {
           "$ref": "#/definitions/ServerAdministrators"
         },
-        "backupConfiguration": {
```
Comment (vishrutshah) [inline arm-analysisservices/2016-05-16/swagger/analysisservices.json]: I see the you are removing the existing property. This potentially will be a breaking change in the generated SDK. 

Breaking changes are not recommended without updating the api-version.

**Review by vishrutshah — CHANGES_REQUESTED** [review]

File: arm-analysisservices/2016-05-16/swagger/analysisservices.json (hunk 1 above)
Comment (taiwu) [inline arm-analysisservices/2016-05-16/swagger/analysisservices.json]: So should I create another json file under the new version folder or directly update the version in this file?

Comment (vishrutshah) [inline arm-analysisservices/2016-05-16/swagger/analysisservices.json]: @taiwu Yes please. Follow the guidelines outlined https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md#filenames-and-folder-structure .

Please make sure you create 2 commits. One with just copy of the old swagger into the new folder and second with the changes to help us review the changes only. Thanks.

Comment (taiwu) [inline arm-analysisservices/2016-05-16/swagger/analysisservices.json]: @vishrutshah could you please take a look at the pull request?
https://github.com/Azure/azure-rest-api-specs/pull/1296

Comment (vishrutshah) [inline arm-analysisservices/2016-05-16/swagger/analysisservices.json]: @taiwu I'd like to close this PR in favor of #1296.

**Pull request closed without merging**